client_cli/src/main.rs

```diff
@@ -111,27 +111,55 @@ pub enum Subcommand {
     Json(json::Args),
 }
 
+/// Context inside which command is executed
+pub trait RunContext {
+    /// Get access to configuration
+    fn configuration(&self) -> &ClientConfiguration;
+    /// Serialize and print data
```

Comment:
```suggestion
    fn configuration(&self) -> &ClientConfiguration;
    
    /// Serialize and print data
```

Reply:
Discussed in DM, let's do it as part of other task